src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/readme.md

```diff
@@ -32,6 +32,20 @@ title: $(service-name)
 subject-prefix: ''
 
 ```
+
+### Directives
+
+> see https://github.com/Azure/autorest/blob/master/docs/powershell/directives.md
+
+``` yaml
+directive:
+# Rename cmdlets with duplicates in their name.
+  - where:
+      subject: ^(OnPremisPublishingProfile)(\1)+
```

Comment:
Missing `e`

Reply:
I'll fix the AutoREST typo via a directive.